Frameworks/include/CoreGraphics/DWriteWrapper.h

```diff
@@ -35,14 +37,17 @@ COREGRAPHICS_EXPORT CFArrayRef _DWriteCopyFontNamesForFamilyName(CFStringRef fam
 COREGRAPHICS_EXPORT CFStringRef _DWriteGetFamilyNameForFontName(CFStringRef fontName);
 
 struct _DWriteFontProperties {
-    DWRITE_FONT_WEIGHT weight;
-    DWRITE_FONT_STRETCH stretch;
-    DWRITE_FONT_STYLE style;
-    CFStringRef familyName;
+    DWRITE_FONT_WEIGHT weight = DWRITE_FONT_WEIGHT_NORMAL;
+    DWRITE_FONT_STRETCH stretch = DWRITE_FONT_STRETCH_NORMAL;
+    DWRITE_FONT_STYLE style = DWRITE_FONT_STYLE_NORMAL;
+
+    woc::unique_cf<CFStringRef> displayName;
+    woc::unique_cf<CFStringRef> postScriptName;
+    woc::unique_cf<CFStringRef> familyName;
 };
 
 // Create DWrite objects
-COREGRAPHICS_EXPORT _DWriteFontProperties _DWriteGetFontPropertiesFromName(CFStringRef fontName);
+extern "C++" std::shared_ptr<_DWriteFontProperties> _DWriteGetFontPropertiesFromName(CFStringRef fontName);
 
```

Comment:
yeah probably.

---
In reply to: [87664812](https://github.com/Microsoft/WinObjC/pull/1366#discussion_r87664812) [](ancestors = 87664812)